### ISSUE TITLE
accessibility add focus state styles to interactive elements. 

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#2646]
We have many elements where there are hover styles but no styles for the focus state it really helps with accessibility specifically for those users who are using a keyboard to navigate the site.

**Comment by nisarhassan12** [comment]
I have made this Pr for it https://github.com/gitpod-io/website/pull/2788 so far I have corrected/added focus state styles for the following:

- `Button` and `LinkButton` components.
- Buttons in the Connect Git repository.

**Comment by nisarhassan12** [comment]
The way to fix this would be to go through the entire site and check all the interactive elements. 

@chrifro Regarding the outline that appears while tabbing through elements on focus is the one that comes with tailwind by default I think we should remove that and have something that best fits our design system.

For instance, https://slack.com/ has its own style for the outline that compliments their design system well

![image](https://user-images.githubusercontent.com/46004116/192142656-43536ac3-9d98-44e4-97ba-0241ef1ee5a2.png)
